src/components/SignUp/OffstageRoles.tsx

```diff
@@ -1,22 +1,263 @@
 import React from 'react';
-import Container from 'react-bootstrap/Container';
+import { fonts } from '../../theme/styleVars';
+import styled from 'styled-components';
+import GenericAccordion from '../../genericComponents/Accordion';
+import Checkbox from '../../genericComponents/Checkbox';
+import yellow_blob_1 from '../../images/yellow_blob_1.svg';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
+import Image from 'react-bootstrap/Image';
+import Container from 'react-bootstrap/Container';
+
+const offstageRolesObj = {
+  general: {
```

Comment:
First step: add new properties to each section object like this:
```
general: {
  sectionStateValue: offstageRolesGeneral,
  sectionStateName: 'offstageRolesGeneral',
```
(Keeping your `textHeader` and `checkboxes` properties, of course. 

You will also need to move this object down into your component so it has access to destructured `formData` properties like `offstageRolesGeneral`